apps/web/modules/event-types/views/event-types-listing-view.tsx

```diff
@@ -373,6 +387,8 @@ export const EventTypeList = ({
       setNativeShare(false);
     }
   }, []);
+  const [shadowList, setShadowList] = useState(types);
```

Comment:
When dragging, everytime a position changes, onReorder returns the updatedList instead of doing it at the end of mouse down event, to track that I am using another list where all changes are made, onMouseDown event we update the main list.